cmd/kubeadm/app/util/user.go

```diff
@@ -0,0 +1,15 @@
+package util
+
+import "os/user"
+
+const (
+	sudoUser = "0"
+)
+
+func IsRoot() (*user.User, bool) {
```

Comment:
@luxas can it be moved to said package in bigger _pre-flight_ PR, which I believe you and @dgoodwin are working on?